dataquality/loggers/model_logger/seq2seq.py

```diff
@@ -135,7 +154,7 @@ def _get_data_dict(self) -> Dict:
             C.epoch.value: [self.epoch] * batch_size,
         }
         if self.split == Split.inference:
-            data["inference_name"] = [self.inference_name] * batch_size
+            data[C.inference_name.value] = [self.inference_name] * batch_size
```

Comment:
Do we need a `.value` here? I think it works simply, too?
```suggestion
            data[C.inference_name] = [self.inference_name] * batch_size
```

Reply:
I did value for consistency with the above 5 lines, but I can do a test removing .value from all of them and push that up

Reply:
Yes we do! This creates a dict that later gets cast to a vaex DataFrame, which can't have enum as the keys to the dict 

The following errors:
```python
from enum import Enum
import vaex

class MyType(str, Enum):
    a = "a"
    b = "b"
    c = "c"

df = vaex.from_dict(
    {
        MyType.a: [1, 2, 3],
        MyType.b: [4, 5, 6],
        MyType.c: [7, 8, 9],
    }
)
```

Reply:
TIL!